source/_integrations/myuplink.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: myUplink
+description: Instructions on how to set up the myUplink integration within Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: '2023.2'
+ha_domain: myuplink
+ha_codeowners:
+  - '@pajzo'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The myUplink integration lets you get information about the devices supporting myUplink using the [official cloud API](https://dev.myuplink.com).
+
+Depending on the type of devices in your system, one or more sensor entities to Home Assistant. Currently, there is support for generic and temperature sensors.
```

Comment:
```suggestion
Depending on the type of devices in your system, one or more sensor entities will be created in Home Assistant. Currently, there is support for generic and temperature sensors.
```

Reply:
Generic sensors sounds weird. Maybe just don't mention what sensors are supported as that will change over time and it's hard to keep the documentation updated to cover it exactly?